prov/efa/src/efa_hmem.c

```diff
@@ -228,6 +237,17 @@ static int efa_domain_hmem_info_init_neuron(struct efa_domain *efa_domain)
 		return 0;
 	}
 
+	/* Neuron currently requires P2P */
+	info->p2p_required_by_impl = true;
+
+	if (ofi_hmem_p2p_disabled()) {
+		info->initialized = true;
```

Comment:
Likewise, `initialized` shouldn't depend on whether or not the user disabled P2P